src/store/import/import.action.js

```diff
@@ -4,13 +4,16 @@ import { IMPORT_ADDED } from './import.reducer';
 
 
 /**
- * Request for fitting a map to a geographic extent
- * @typedef {Object} ImportOption
+ * Properties for a import of vectorDataResources
```

Comment:
Not necessarily vector data,  could also be  a WMS URL or something else in the future